.github/ISSUE_TEMPLATE/library-request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: cdnjs Library Request
+about: Request a new library to be added to cdnjs
+title: "Add: xxx"
+labels: "🏷 Library Request"
+---
+
+<!-- Please fill out the information in this issue template so that we can
+efficiently process your request for a new library to be added to cdnjs -->
+
+<!-- To add a library to cdnjs, it must have a way for an auto-update mechanism
+to be used. This can either be via a Git repository that has versions tagged,
+with the required files included in the repository at the tag. Or, this can be
```

Comment:
I don't know about this `Or, ` here, it feels like this should be `Alternatively, ` personally.